protostar/configuration_file/configuration_file.py

```diff
@@ -106,6 +106,7 @@ def __init__(self, contract_name: str, expected_declaration_location: str):
         super().__init__(
             f"Couldn't find `{contract_name}` in `{expected_declaration_location}`"
         )
+        self.contract_name = contract_name
```

Comment:
what is this for?

Reply:
I'm re-raising a `CheatcodeException` from `ContractNameNotFoundException` in declare hint local. I need a contract name for the exception message.

Reply:
right, thanks